src/util/FactorUtil.js

```diff
@@ -139,6 +145,9 @@ function (Okta, TimeUtil) {
     if (provider === 'OKTA' && factorType === 'push') {
       return 'OKTA_VERIFY_PUSH';
     }
+    if (factorType === 'token:software:hotp') {
```

Comment:
I think we could put this in getFactorNameForFactorType. Webauthn is there also

Reply:
yeah, I was going to mention the same thing. This function goes through all the provider/factorType combinations, and `getFactorNameForFactorType` looks only at factorTypes, which is what we have here